x-pack/plugins/security_solution/public/detection_engine/rule_gaps/logic/use_schedule_rule_run.ts

```diff
@@ -15,18 +16,22 @@ import * as i18n from '../translations';
 export function useScheduleRuleRun() {
   const { mutateAsync } = useScheduleRuleRunMutation();
   const { addError, addSuccess } = useAppToasts();
+  const { telemetry } = useKibana().services;
 
   const scheduleRuleRun = useCallback(
     async (options: ScheduleBackfillProps) => {
       try {
         const results = await mutateAsync(options);
+        telemetry.reportManualRuleRunExecute({
+          rangeInMs: options.timeRange.endDate.diff(options.timeRange.startDate),
+        });
```

Comment:
This will only capture successful manual executions, do we need to capture failures as well?